packages/scripts/utils/config.js

```diff
@@ -234,7 +234,10 @@ function getWebpackEntryPoints() {
 
 						// Detects the proper file extension used in the `src` directory.
 						const [ entryFilepath ] = glob(
-							`src/${ entryName }.[jt]s?(x)`,
+							`src/${ entryName }.[jt]s?(x)`.replace(
```

Comment:
Should the replace get applied to the entry name?

Reply:
I did this because I did not understand the whole wp-script and entryName was used as a key for the subsequent accumulator.
However, the following method of replacing `entryName` also worked for me.

```javascript
const entryName = filepath
	.replace( extname( filepath ), '' )
	.replace( srcDirectory, '' )
	.replace( /\\/g, '/' );
```

I followed the code and found that the getWebpackEntryPoints function is only used in the entry property of webpack, so I don't think it affects anything else.
I also think it would be cleaner to replace the `entryName`, so I will modify it that way if necessary.